rpc/master/service_client.go

```diff
@@ -35,6 +35,11 @@ type WaitServiceRequest struct {
 	Recursive  bool
 }
 
+type EvaluateServiceRequest struct {
```

Comment:
I think these requests objects make more sense in the *_server.go file:
see https://github.com/control-center/serviced/blob/12b81313b3eec2f3d6353a6ec6733299a2a5fcb6/rpc/master/instance_server.go#L32-L41